ui-lib/src/main/java/co/electriccoin/zcash/ui/common/repository/AddressBookRepository.kt

```diff
@@ -263,6 +283,23 @@ class AddressBookRepositoryImpl(
         }
     }
 
+    private suspend fun getAddressBookKey(): AddressBookKey {
+        val key = addressBookKeyStorageProvider.getAddressBookKey()
```

Comment:
This will eventually need to be keyed by whatever identifier we use to distinguish between accounts. Currently that is `Account` (being accounts under the same seed phrase); in future it may also need to accommodate imported UFVKs. I could not figure out how to do this, so I'll leave it to @HonzaR or @Milan-Cerovsky to either do this now, or be prepared to migrate the storage provider to enable this later.

Reply:
We will do this as a part of multi-account support for keystone

Reply:
sure why not